### PR TITLE
Jenifar fathima h

### PR DESCRIPTION
**Pull request by Jenifar-fathima**

### CODE REVIEW

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+using System;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        string word = Console.ReadLine();
```
Comment (msindia-learning) [inline JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs]: # Naming Rules:

1. Integer - nVariableName - **Prefix n**
2. Float - fVariableName - **Prefix f**
3. Double - dVariableName - **Prefix d**
4. Char- cVariableName - **Prefix c**
5. String - strVariableName - **Prefix str**
6. Boolean- bVariableName - **Prefix b**
7. Array - lstVariableName - **Prefix lst**

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        bool tells =Hello(word);
```
Comment (msindia-learning) [inline JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs]: ### Name must be meaningful.

e.g.., HasContainsHello - **Method name should explain your code**

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            Console.WriteLine("YES");
+        }
+        else
+        {
+            Console.WriteLine("NO");
+        }
+    }
+    public static bool Hello(string word)
+    {
+        int hx = word.IndexOf('h');
```
Comment (msindia-learning) [inline JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs]: ### Don't use inbuilt functions

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/question.png
Comment (msindia-learning) [inline JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/question.png:1]: ### Rename this file - Question.png

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Stones on the Table/Stonetable.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+using System;
+ 
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        int n = int.Parse(Console.ReadLine());
```
Comment (msindia-learning) [inline JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Stones on the Table/Stonetable.cs]: ### Update naming rules

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Wrong Subtraction/WrongSub.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
+using System;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
```
Comment (msindia-learning) [inline JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Wrong Subtraction/WrongSub.cs:6]: Update naming rules

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs (hunk 1 above)
Comment (Jenifar-fathima) [reply JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs]: Done

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs (hunk 2 above)
Comment (Jenifar-fathima) [reply JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs]: Done

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs (hunk 3 above)
Comment (Jenifar-fathima) [reply JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Chat room/Chatroom.cs]: Done

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Stones on the Table/Stonetable.cs (hunk 4 above)
Comment (Jenifar-fathima) [reply JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Stones on the Table/Stonetable.cs]: done

File: JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Wrong Subtraction/WrongSub.cs (hunk 5 above)
Comment (Jenifar-fathima) [reply JenifarFathimaH/00 - Projects/03 - C#/Practice Problems/A. Wrong Subtraction/WrongSub.cs:6]: done

**Review by msindia-learning — APPROVED** [review]

**Pull request merged**